charts/scalardl/values.yaml

```diff
@@ -9,6 +9,10 @@ nameOverride: ""
 fullnameOverride: ""
 
 envoy:
+  # -- enable envoy
+  enabled: true
+  # -- String to partially override envoy.fullname template
+  nameOverride: ledger
```

Comment:
I feel the current naming looks a bit inconsistent.
`scalardl` chart has envoy and ledger so I think they should be named as `scalardl-envoy` and `scalardl-ledger` respectively.
(For `scalardl-audit` chart, its envoy and auditor seem to be named properly as `scalardl-audit-envoy` and `scalardl-audit-auditor`)

To make it happen, seems like `nameOverride` has to be empty string but is it possible?


```
scalardl-audit-auditor-6886c85d56-brdv7   1/1     Running     0          139m
scalardl-audit-auditor-6886c85d56-wdpjr   1/1     Running     0          139m
scalardl-audit-auditor-6886c85d56-z8lt5   1/1     Running     0          139m
scalardl-audit-envoy-7c64bff78d-f4gl8     1/1     Running     0          139m
scalardl-audit-envoy-7c64bff78d-n7rps     1/1     Running     0          139m
scalardl-audit-envoy-7c64bff78d-nsd68     1/1     Running     0          139m
scalardl-ledger-7f889464dd-5mlj5          1/1     Running     0          142m
scalardl-ledger-7f889464dd-bf9zd          1/1     Running     0          142m
scalardl-ledger-7f889464dd-gkggb          1/1     Running     0          142m
scalardl-ledger-envoy-696ff46844-lwm9w    1/1     Running     0          142m
scalardl-ledger-envoy-696ff46844-n2g6l    1/1     Running     0          142m
scalardl-ledger-envoy-696ff46844-r5d8t 
```

Reply:
I'lll adjust the name to see if it can be made into the name you have in mind. :bow:

Reply:
@feeblefakie 

It has been fixed.
Is this the correction that you had in mind?

PTAL

https://github.com/scalar-labs/helm-charts/pull/29/commits/f141abde9ff81a4d87bf5a5320cec9ff31cc2f9b

``` bash
$ kubectl get pods
NAME                                      READY   STATUS      RESTARTS   AGE
scalardl-audit-auditor-6886c85d56-6hlfw   1/1     Running     0          41m
scalardl-audit-auditor-6886c85d56-gt4bn   1/1     Running     0          41m
scalardl-audit-auditor-6886c85d56-pr8vw   1/1     Running     0          41m
scalardl-audit-envoy-7c64bff78d-9c8rc     1/1     Running     0          41m
scalardl-audit-envoy-7c64bff78d-jtnzg     1/1     Running     0          41m
scalardl-audit-envoy-7c64bff78d-nsd5j     1/1     Running     0          41m
scalardl-envoy-6c7dbdbbc-7xzrj            1/1     Running     0          41m
scalardl-envoy-6c7dbdbbc-nwgxd            1/1     Running     0          41m
scalardl-envoy-6c7dbdbbc-tx7d9            1/1     Running     0          41m
scalardl-ledger-7f889464dd-5j4hv          1/1     Running     0          41m
scalardl-ledger-7f889464dd-8hcrg          1/1     Running     0          41m
scalardl-ledger-7f889464dd-tqk25          1/1     Running     0          41m
```

Reply:
``` sh
$ helm upgrade --install my-relase-scalardl charts/scalardl --namespace default -f ./.conf/scalardl-custom-values.yaml
$ helm upgrade --install my-relase-scalardb charts/scalardb --namespace default -f ./.conf/scalardb-custom-values.yaml
$ helm upgrade --install my-relase-scalardl-audit charts/scalardl-audit --namespace default -f ./.conf/scalardb-custom-values.yaml
```
``` sh
$ kubectl get pods
NAME                                                 READY   STATUS      RESTARTS   AGE
load-schema-schema-loading-2tckb                     0/1     Completed   0          17m
my-release-scalardb-8fd7c4846-877nz                  1/1     Running     0          5m16s
my-release-scalardb-8fd7c4846-nxppk                  1/1     Running     0          5m16s
my-release-scalardb-8fd7c4846-sq9kx                  1/1     Running     0          5m16s
my-release-scalardb-envoy-6d6b7b9cd9-c6rmc           1/1     Running     0          5m16s
my-release-scalardb-envoy-6d6b7b9cd9-g22f9           1/1     Running     0          5m16s
my-release-scalardb-envoy-6d6b7b9cd9-xrqnt           1/1     Running     0          5m16s
my-release-scalardl-audit-auditor-7d95bcdd6f-b25vg   1/1     Running     0          4m39s
my-release-scalardl-audit-auditor-7d95bcdd6f-dkxpn   1/1     Running     0          4m40s
my-release-scalardl-audit-auditor-7d95bcdd6f-sjvqp   1/1     Running     0          4m40s
my-release-scalardl-audit-envoy-6b65c8b6d-bzqmf      1/1     Running     0          4m39s
my-release-scalardl-audit-envoy-6b65c8b6d-h6k6c      1/1     Running     0          4m39s
my-release-scalardl-audit-envoy-6b65c8b6d-jdztb      1/1     Running     0          4m40s
my-release-scalardl-envoy-f688b8c99-c4qct            1/1     Running     0          5m51s
my-release-scalardl-envoy-f688b8c99-hmq8j            1/1     Running     0          5m51s
my-release-scalardl-envoy-f688b8c99-jdkcn            1/1     Running     0          5m51s
my-release-scalardl-ledger-649cb5fc95-6dmsk          1/1     Running     0          5m51s
my-release-scalardl-ledger-649cb5fc95-6fd2x          1/1     Running     0          5m51s
my-release-scalardl-ledger-649cb5fc95-t2txx          1/1     Running     0          5m51s
```